### DISCUSSION TITLE
Hide NSFW categories when NSFW is disabled

### DISCUSSION THREAD

**Discussion by skittlz444** [civitai/civitai#386] [Ideas]
I recently did a presentation on SD for my workplace, and wanted to show CivitAI as a great site for finding new and exciting models. However I decided heavily against it when I realised that even though NSFW images and tags are hidden when NSFW is disabled, the NSFW categories like "Porn" and "Hentai" still show in the categories component on the home page. 
Would it be possible to hide these as well, as people who don't want NSFW content obviously don't want those categories either.

**Comment by JohnMadden123456** [comment]
I agree, that's too much to handle for grown up man and woman. We allowed to vote, buy weapon and kill other people with it, we allowed to start a war with another country, but reading non-safe words like n-words, "porn" or "hentai" is too much. We should start burning books and establishing true christian halifate. Also please ban all women depiction and force everone to generate models wearing burka.

**Reply by skittlz444** [reply]
Thanks for the sarcastic response, do you mind doing being quiet for a second while I give a presentation to the ceo of a multinational company and a group of other higher ups with the word hentai on screen? Already hard enough to convince credibility and the potential of the tech to some people.

If there's a toggle for NSFW content, then it should cover all NSFW content, categories included. I'm just pointing out that those were missed. No one's forcing you to have the filer on.

**Comment by JustMaier** [comment]
Hey @skittlz444 totally agree with you. Looks like our logic for filtering out certain tags based on your content preferences wasn't working quite right. We should have that resolved soon.

I apologize for the delay

**Comment by JustMaier** [comment]
Ok @skittlz444 this has been fixed.

**Reply by skittlz444** [reply]
Brilliant thanks, confirmed working as guest, logged in user in safe mode, and via my filters. Also confirmed that the tags still correctly appear in scenarios where they should.

Can you tell I'm a QA tester? 😂

**Reply by JustMaier** [reply]
Haha. Can you tell we don't have any QA testers on our team? 🤣
